src/dumbo/src/ns.rs

```diff
@@ -363,15 +389,18 @@ mod tests {
     #[test]
     #[allow(clippy::cognitive_complexity)]
     fn test_ns() {
-        let mut ns = MmdsNetworkStack::new_with_defaults();
+        let mut ns = MmdsNetworkStack::new_with_defaults(vec![Ipv4Addr::from(DEFAULT_IPV4_ADDR)]);
```

Comment:
If you want to check the default scenario, you should pass as argument an empty `Vec` here, not one that already contains the default value. Also, for testing `new_with_defaults()` function properly, we should also call this function with some non-empty `Vec<IpvdAddr>` as parameter.